src/ladds/io/SatelliteLoader.cpp

```diff
@@ -84,32 +84,25 @@ void SatelliteLoader::loadSatellites(AutoPas_t &autopas, ConfigReader &config, c
 
 std::vector<Constellation> SatelliteLoader::loadConstellations(ConfigReader &config, const Logger &logger) {
   std::vector<Constellation> constellations;
-  auto coefficientOfDrag = config.get<double>("sim/prop/coefficientOfDrag");
   auto constellationList = config.get<std::string>("io/constellationList", "", true);
-  // altitudeSpread = 3 * sigma , altitudeDeviation = sigma (= standardDeviation)
-  auto altitudeDeviation = config.get<double>("io/altitudeSpread", 0.0) / 3.0;
   if (not constellationList.empty()) {
     const auto insertionFrequency = config.get<int>("io/constellationFrequency", 1);
     auto constellationDataStr = config.get<std::string>("io/constellationList");
     // count constellation by counting ';'
-    int nConstellations = 1;
-    for (char con : constellationDataStr) {
-      if (con == ';') {
-        nConstellations++;
-      }
-    }
+    int nConstellations = std::count(constellationDataStr.begin(), constellationDataStr.end(), ';') + 1;
```

Comment:
Why not expect constellations as an actual List/Sequence in `yaml`? Like e.g. `[AstraPhase1, OneWeb]`. However, if this complicates things in the `ConfigReader::get` function I'm fine with keeping the string based approach.

Reply:
I was not able to find out how to get a list using `ConfigReader::get`. Is it possible and I just didn't find out?

Reply:
I don't know that's why I was asking :) Ok then we just keep it as it is.